runtime-sdk/src/storage/current.rs

```diff
@@ -0,0 +1,514 @@
+//! A store attached to the current thread.
```

Comment:
This looks ok :+1: (i checked this a bit more in depth, didn't fully check all places that were updated that use storage, but seems a straightforward update in most places)